test/src/specs/relay/block_relay.rs

```diff
@@ -1,29 +1,11 @@
-use crate::node::{connect_all, exit_ibd_mode};
+use crate::util::mining::mine;
+use crate::util::mining::out_ibd_mode;
 use crate::utils::{now_ms, sleep, wait_until};
 use crate::{Node, Spec};
 use ckb_types::prelude::*;
 use log::info;
 use std::time::Duration;
 
-pub struct BlockRelayBasic;
```

Comment:
why was it removed?

Reply:
This case is good for the early stage. However, I think it is too simple for now, so I removed it when I resolve PR conflicts.